gpkit/keydict.py

```diff
@@ -116,6 +116,12 @@ def __setitem__(self, key, value):
                 emptyvec = np.full(key.shape, np.nan, **kwargs)
                 dict.__setitem__(self, key, emptyvec)
         for key in self.keymap[key]:
+            if getattr(value, "exp", None) is not None and not value.exp:
```

Comment:
Yup. For varkeys in particular. :-/ they still have `None` attributes for everything...